core/src/main/java/org/apache/druid/common/config/ConfigSerde.java

```diff
@@ -25,5 +25,6 @@
 {
   byte[] serialize(T obj);
   String serializeToString(T obj);
+  String serializeSkipNullToString(T obj);
```

Comment:
I don't understand this part of the code so well, could you add a javadoc and talk about what the expectations of this function are and how they're different from  `serializeToString` above. Maybe javadocs will help clarify this for me

Reply:
Done